provider/scripts/spark/offline_store_spark_runner.py

```diff
@@ -71,8 +71,12 @@ def display_data_metrics(df, spark):
     if len(rows_as_list) > 100:
         rows_as_list = rows_as_list[:100]
     for row in rows_as_list:
-        row_list = row.asDict().values()
-        rows.append(list(row_list))
+        row_values = list(row.asDict().values())
+        for index in range(len(row_values)):
+            curr_value = row_values[index]
+            if isinstance(curr_value, datetime.datetime):
+                row_values[index] = curr_value.strftime("%m/%d/%Y")
```

Comment:
Is this the highest precision it can have? Should try to maintain the given precision

Reply:
yeah I can add the time itself. one sec.